src/components/NavBar/AccountMenu.tsx

```diff
@@ -40,7 +40,7 @@ const AccountMenu = () => {
         <Avatar />
         <Stack sx={styles.nameContainer}>
           <Typography>{user.displayName}</Typography>
-          <Typography>{user.username}</Typography>
+          <Typography>{`@${user.username}`}</Typography>
```

Comment:
Nothing to do with this change here, but I think we should remove the ripple from this Button component. Would look cleaner